odl/solvers/iterative/statistical.py

```diff
@@ -121,6 +138,131 @@ def mlem(op, x, data, niter=1, noise='poisson', callback=None):
         raise RuntimeError('unknown noise model')
 
 
+def osmlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
+    """Ordered Subsets Maximum Likelihood Expectation Maximation algorithm.
+
+    Attempts to solve::
+
+        max_x L(data | x)
```

Comment:
Flipped. This is the pure ML problem, but despite its name, the MLEM method tries to maximize the likelihood of the unknown parameters x given the data. See for example [this paper](http://link.springer.com/chapter/10.1007/978-3-319-01712-9_2).